web/app/js/components/ServiceMesh.jsx

```diff
@@ -32,9 +32,15 @@ const componentNames = {
   "proxy-api":    "Controller proxy-api",
   "public-api":   "Controller public-api",
   "tap":          "Controller tap",
-  "telemetry":    "Controller telemetry",
+  "telemetry":    "Controller Telemetry",
   "web":          "Web UI"
 };
+
+const componentGraphTitles = {
+  "destination":  "Service lookups",
```

Comment:
Do we still need a "destination" key in this map, since we're no longer graphing traffic to the destination service?